docs/getting-started/install/linux/index.md

```diff
@@ -133,45 +138,116 @@ Alpha update channel
 
 If you want to test bleeding-edge updates to Mono, we now have an alpha update channel. Please report bugs promptly, so we have time to fix them before packages are migrated to stable!
 
-#### Debian, Ubuntu, and derivatives (alpha channel)
```

Comment:
we may want to consider splitting alpha/beta to their own page as it gets pretty long